### ISSUE TITLE
Unable to install jupyter notebook

### ISSUE THREAD

**Issue by Tomar-shubham** [jupyter/help#277]
Hi,
Getting the following error if i try to install jupyter notebook on my command prompt or Anaconda prompt.Please help
![cmd](https://user-images.githubusercontent.com/35186887/34647354-be6b7aaa-f3a6-11e7-9900-3be606a285e1.JPG)

**Comment by takluyver** [comment]
Do the files it mentions in the error message exist?

**Comment by Tomar-shubham** [comment]
No, in place of python.exe i have pythonw.exe present in my Anaconda3 folder upon installation.

**Comment by takluyver** [comment]
You should have both `python.exe` and `pythonw.exe`. I suggest removing Anaconda entirely and installing it again.

**Comment by Tomar-shubham** [comment]
Tried re-installing several times but issue still exists.

**Comment by takluyver** [comment]
And `python.exe` still doesn't exist? Try filing an issue here: https://github.com/ContinuumIO/anaconda-issues/ - it sounds like an install problem with Anaconda.
